### DISCUSSION TITLE
NEXTAUTH_URL and NEXTAUTH_SECRET seem to be required but isn't mentioned in getting started doc

### DISCUSSION THREAD

**Discussion by nabeelvalley** [nextauthjs/next-auth#6673] [Help]
Not sure if I'm doing something wrong but I was working through the Next.js example [here](https://next-auth.js.org/getting-started/example) and [here](https://github.com/nextauthjs/next-auth-example) and it seems like I couldn't get either to run until I defined the `NEXTAUTH_URL` and `NEXTAUTH_SECRET` variables - in the doc it only mentions that the `NEXTAUTH_URL` is required for production deployments

**Comment by devdatkumar** [comment]
NEXTAUTH_URL and NEXTAUTH_SECRET have been inferred since v4.

https://authjs.dev/reference/nextjs
